app/views/application/_header.html.erb

```diff
@@ -0,0 +1,26 @@
+<nav class="navbar navbar-expand-lg navbar-dark custom-navbar">
+  <div class="container-fluid">
+    <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbarSupportedContent" aria-controls="navbarSupportedContent" aria-expanded="false" aria-label="Toggle navigation">
+      <span class="navbar-toggler-icon"></span>
+    </button>
+
+      <%= link_to(current_page?(root_path) ? "Robert's Budget" : 'Transactions', root_path, class: 'navbar-brand') %>
+    <form class="d-flex" role="search">
+      <svg xmlns="http://www.w3.org/2000/svg" width="20" height="20" fill="white" class="bi bi-search" viewBox="0 0 15 15">
+        <path d="M11.742 10.344a6.5 6.5 0 1 0-1.397 1.398h-.001c.03.04.062.078.098.115l3.85 3.85a1 1 0 0 0 1.415-1.414l-3.85-3.85a1.007 1.007 0 0 0-.115-.1zM12 6.5a5.5 5.5 0 1 1-11 0 5.5 5.5 0 0 1 11 0"/>
+      </svg>
+    </form>
+
+    <div class="collapse navbar-collapse" id="navbarSupportedContent">
+      <ul class="navbar-nav me-auto mb-2 mb-lg-0">
+        <br/>
+        <% if user_signed_in? %>
+            <%= button_to 'Sign out', destroy_user_session_path, method: :delete, class: ['btn', 'btn-outline-light'] %>
+          <% else %>
+            <li class="nav-item">  <%= link_to 'Login', user_session_path, class: 'nav-link' %></li>
+            <li class="nav-item">  <%= link_to 'Sign up', new_user_registration_path, class: ['nav-link'] %></li>
+          <% end %>
+      </ul>
+    </div>
+  </div>
+</nav>
```

Comment:
- [ ] kindly consider including the name of the app on splash screen and while doing so make sure you stick to the given design of page